From: SZALAY Attila <sasa@debian.org>
Date: Sat, 3 Sep 2016 14:59:41 +0200
Subject: Fixed line-braking issue

The list of automatically loaded modules contained
no space and therefore the man parser would not able
add line break if needed.

Signed-off-by: SZALAY Attila <sasa@debian.org>
---
 doc/man/syslog-ng.8.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/man/syslog-ng.8.xml b/doc/man/syslog-ng.8.xml
index 35ad1d8..74b9b46 100644
--- a/doc/man/syslog-ng.8.xml
+++ b/doc/man/syslog-ng.8.xml
@@ -104,7 +104,7 @@
             </indexterm>
           </term>
           <listitem>
-            <para>A comma-separated list of the modules that are loaded automatically. Modules not loaded automatically can be loaded by including the <parameter>@module &lt;modulename&gt;</parameter> statement in the  configuration file. The following modules are loaded by default: . Available only in  and later.</para>
+            <para>A comma-separated list of the modules that are loaded automatically. Modules not loaded automatically can be loaded by including the <parameter moreinfo="none">@module &lt;modulename&gt;</parameter> statement in the syslog-ng OSE configuration file. The following modules are loaded by default: <parameter condition="ose" moreinfo="none">affile, afprog, afsocket, afuser, basicfuncs, csvparser, dbparser, syslogformat, afsql</parameter>. Available only in <phrase condition="ose">syslog-ng Open Source Edition 3.3</phrase> and later.</para>
           </listitem>
         </varlistentry>
         <varlistentry>
